repair/featurize/featurize.py

```diff
@@ -13,6 +13,8 @@ def __init__(self, dataset, env, featurizers):
         for f in featurizers:
             f.setup_featurizer(self.ds, self.total_vars, self.classes, self.processes)
         tensors = [f.create_tensor() for f in featurizers]
+        # save size info of all featurizers
+        self.featurizer_info = [(str(type(featurizers[i])), t.cpu().numpy().shape[2]) for i, t in enumerate(tensors)]
```

Comment:
why isn't t.size() enough? I don't think we need t.cpu().numpy().shape[2] we can just use t.size() (similar to shape for numpy arrays)

Reply:
changed